core/scripts/utils/sort-translations.js

```diff
@@ -0,0 +1,19 @@
+const { readFileSync, writeFileSync } = require('fs');
+const { EOL } = require('os');
+
+const fixTranslation = (filename) => {
+  const content = readFileSync(filename, 'utf8');
```

Comment:
While you are reading mostly CSV files, create file stream, while translation files may be huge.

To properly sort them and use transform pipe, then pipe it to write stream. 
IMO this faster solution, easier to maintain. 

And I'm not sure why `sort-translations.js` contains function `fixTranslation`, it may make some mess